Suggested changes for ARM docs (#558)
stackabletech/documentation: modules/concepts/pages/arm64-support.adoc

:description: This page describes how to access ARM64 based SDP early
:keywords: Multi-Architecture, infrastructure, docker, image, tags, early-access

WARNING: This is an experimental feature of the Stackable Data Platform (SDP) and still under https://github.com/stackabletech/issues/issues/463[development].

> == Early Access

Reviewer: ```suggestion
== Early access
```

SDP release 23.11.0 has been ported to the ARM64 architecture.
These images are stored within the `stackable-experimental` organization in our https://repo.stackable.tech/#browse/browse:docker:v2%2Fstackable-experimental[Docker repository].

> They are not available by default, so you need to switch to the `stackable-experimental` organization before pulling them for use with the data platform.

Reviewer 2: ```suggestion
They are not available by default, so you need to switch to the `stackable-experimental` organization before pulling them.
E.g.
[source,bash]
----
docker pull docker.stackable.tech/stackable-experimental/secret-operator:23.11.0
----
```

> === Available Releases

Reviewer: ```suggestion
=== Available releases
```
use sentence case in headings

By declaring `--set image.repository=docker.stackable.tech/stackable-experimental/commons-operator` you will overwrite the image selected by default with one located in the organization `stackable-experimental`. The other operators can be installed in the same way.

> TODO: Change this

Reviewer 2: ```suggestion

```

> WARNING: You have to install every operator you need for a given demo individually with helm using `--skip-release` with stackablectl. Otherwise, it will try to install x86 operators and pull ARM64 product images afterwards.

Reviewer 2: ```suggestion
WARNING: You have to install every operator you need for a demo individually with helm. Then installing the specific demo using `--skip-release` with `stackablectl`. Otherwise, `stackablectl` will try to install x86 operators.
```

Author: This does not help me understand. I made a different proposal.

> === Prepared Demos

Reviewer: ```suggestion
=== Prepared demos
```
